packages/urlstate/useUrlStateBase/README.md

```
@@ -52,3 +53,10 @@ Updates both the state and the URL.
 - `value?: T | Partial<T> | (currState: T) => T` - Optional new state value or a function that receives the current state and returns the new state.
 - `options?: Options` - Optional options object. When `replace` is true it will use router.replace. Other nextjs native options for `router`'s push/replace.
 
+## `reset`
+
+Reset state and URL to default
+
+### Parameters:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)